### ISSUE TITLE
Support for ngrx/store 12

### ISSUE THREAD

**Issue by tarlepp** [btroncone/ngrx-store-localstorage#185]
I just noticed this;
```bash
node@4999cd08574c:/app$ ng update
Using package manager: 'yarn'
Collecting installed dependencies...
Found 67 dependencies.
    We analyzed your package.json, there are some packages to update:
    
      Name                                     Version                  Command to update
     --------------------------------------------------------------------------------------
      @ngrx/store                              11.1.1 -> 12.0.0         ng update @ngrx/store
    
    There might be additional packages which don't provide 'ng update' capabilities that are outdated.
    You can update the additional packages by running the update command of your package manager.
node@4999cd08574c:/app$ ng update @ngrx/store
Using package manager: 'yarn'
Collecting installed dependencies...
Found 67 dependencies.
Fetching dependency metadata from registry...
                  Package "ngrx-store-localstorage" has an incompatible peer dependency to "@ngrx/store" (requires "^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0", would install "12.0.0").
✖ Migration failed: Incompatible peer dependencies found.
Peer dependency warnings when installing dependencies means that those dependencies might not work correctly together.
You can use the '--force' option to ignore incompatible peer dependencies and instead address these warnings later.
  See "/tmp/ng-8vqa7b/angular-errors.log" for further details.

node@4999cd08574c:/app$
```

**Comment by Ruslancic** [comment]
Same here +1!

**Comment by Ic4r0** [comment]
Same +1

**Comment by BBlackwo** [comment]
:tada: This issue is resolved in version 12.0.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/ngrx-store-localstorage/v/12.0.0)
- [GitHub release](https://github.com/btroncone/ngrx-store-localstorage/releases/tag/v12.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
